src/SFA.DAS.ProviderRegistrations/Services/TrainingProviderApiClient.cs

```diff
@@ -0,0 +1,65 @@
+﻿using Microsoft.Extensions.Logging;
+using Newtonsoft.Json;
+using SFA.DAS.ProviderRegistrations.Configuration;
+using SFA.DAS.ProviderRegistrations.Models;
+using System.Net;
+using System.Net.Http;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.ProviderRegistrations.Services
+{
+    ///<inheritdoc cref="ITrainingProviderApiClient"/>
+    public class TrainingProviderApiClient : ITrainingProviderApiClient
+    {
+        private readonly HttpClient _httpClient;
+        private readonly TrainingProviderApiClientConfiguration _config;
+        private readonly ILogger<TrainingProviderApiClient> _logger;
+
+        public TrainingProviderApiClient(
+            ITrainingProviderApiClientFactory factory,
+            TrainingProviderApiClientConfiguration config,
+            ILogger<TrainingProviderApiClient> logger)
+        {
+            _httpClient = factory.CreateHttpClient();
+            _config = config;
+            _logger = logger;
+        }
+
+        ///<inherit-doc />
+        public async Task<GetProviderSummaryResult> GetProviderDetails(long providerId)
+        {
+            _logger.LogInformation("Getting Training Provider Details for ukprn:{0} returned OK", providerId);
+
+            var url = $"{BaseUrl()}api/providers/{providerId}";
+
+            var requestMessage = new HttpRequestMessage(HttpMethod.Get, url);
+
+            var response = await _httpClient.SendAsync(requestMessage).ConfigureAwait(false);
+
+            switch (response.StatusCode)
+            {
+                case HttpStatusCode.OK:
+                    _logger.LogInformation("{Url} returned OK", url);
+                    var json = await response.Content.ReadAsStringAsync().ConfigureAwait(false);
+                    return JsonConvert.DeserializeObject<GetProviderSummaryResult>(json);
+                case HttpStatusCode.NotFound:
+                    _logger.LogInformation("{Url} returned not found status code", url);
+                    return default;
+                default:
+                    _logger.LogError("{Url} returned unexpected status code", url);
```

Comment:
Not sure we need this logging, not sure it would help with any problem diagnosis that wasn't in the other API

Reply:
Removed the logger from TrainingProviderApiClient.cs